Error with topojson on npm install

Hi, I have the following error when installing Turf 6.2.0 with npm install:

npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /app/node_modules/topojson/node_modules/topojson-server/bin/geo2topo
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/app/node_modules/topojson/node_modules/topojson-server/bin/geo2topo'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

I have no issues when installin with yarn install.
Tried with both node 14 and 15.
I'm on macos, but I get the same error when building my application with docker.

I have the same issue here trying to upgrade to 6.3.0.
Running Node v15.6.0 on Ubuntu 20.04.
```
❯ npm install @turf/turf@latest
npm WARN deprecated topojson@3.0.2: Use topojson-client, topojson-server or topojson-simplify directly.
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /xxx/xxx/xxx/node_modules/topojson/node_modules/topojson-server/bin/geo2topo
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/xxx/xxx/xxx/node_modules/topojson/node_modules/topojson-server/bin/geo2topo'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/xxx/.npm/_logs/2021-01-22T15_50_26_409Z-debug.log
```
I also see the file in `node_modules/topojson/node_modules/topojson-server/bin/geo2topo`, but the install won't work.  
I've tried removing all the packages and starting clean, but still no joy.

I can selectively pick out the modules I need individually so not so pressing for me.

Thank you for the great library, my work would not be possible without it.

--- a/packages/turf-concave/lib/turf-polygon-dissolve.ts
+++ b/packages/turf-concave/lib/turf-polygon-dissolve.ts
@@ -8,7 +8,8 @@ import {
 } from "@turf/helpers";
 import { getType } from "@turf/invariant";
 import { flattenEach } from "@turf/meta";
-import { merge, topology } from "topojson";
+import { merge } from "topojson-client";
+import { topology } from "topojson-server";
 
 /**
  * Dissolves all overlapping (Multi)Polygon

--- a/packages/turf-concave/package.json
+++ b/packages/turf-concave/package.json
@@ -74,6 +74,7 @@
     "@turf/invariant": "^6.3.0",
     "@turf/meta": "^6.3.0",
     "@turf/tin": "^6.3.0",
-    "topojson": "3.x"
+    "topojson-client": "3.x",
+    "topojson-server": "3.x"
   }
 }

